package.json

```diff
@@ -27,6 +27,11 @@
     "test:testem": "testem -f testem.dist.json"
   },
   "dependencies": {
+    "@glimmer/compiler": "^0.21.1",
+    "@glimmer/node": "^0.21.1",
+    "@glimmer/reference": "^0.21.0",
```

Comment:
No I don't believe so. The way `lerna publish` works is that it looks if packages were effected in between the `HEAD` and the last tag if its not effected it will not publish a new version.